src/regular/jsonrepair.ts

```diff
@@ -211,6 +218,40 @@ export function jsonrepair(text: string): string {
     return false
   }
 
+  /**
+   * Skip markdown json wrapper like ```json or ```
+   */
+  function skipMarkdownJsonWrapper(): boolean {
```

Comment:
One more remark: can you rename the function `skipMarkdownJsonWrapper` into `skipMarkdownCodeBlock` and update the comment accordingly? I think that is the correct name for the tripple backticks.